lnrpc/routerrpc/router_server.go

```diff
@@ -740,10 +744,57 @@ func (s *Server) trackPayment(identifier lntypes.Hash,
 	case err != nil:
 		return err
 	}
+
+	// Stream updates to the client
+	return trackPaymentStream(
+		s,
+		router,
+		subscription,
+		noInflightUpdates,
+		stream.Send,
+		stream.Context,
+		fmt.Sprintf("Payment stream %v canceled", identifier),
```

Comment:
I think this log can also be done outside the call by checking the return value of `trackPaymentStream` for `context.Done`